log/file_handler.ts

```diff
@@ -54,13 +57,24 @@ export class FileHandler extends BaseHandler {
       truncate: this._mode !== "a",
       write: true,
     };
+    this.#isOpen = false;
   }
 
-  override setup() {
+  open() {
+    if (this.#isOpen) return;
```

Comment:
This isn't relevant to this PR. I might argue it's unnecessary, as we have the open/close status of `this._file` we can use instead.